src/components/Node/__tests__/Node.test.tsx

```diff
@@ -11,7 +11,7 @@ describe('<Node />', () => {
   it('renders a node with text, icon, and correct styles', () => {
     const { container, getByText, getByRole } = render(
       <Node
-        pathwayState={testState}
+        pathwayNode={testState}
```

Comment:
```suggestion
        pathwayNode={testNode}
```

rename on line 5 too